Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -196,6 +187,8 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
   }
 
   private func bindProjectPageViewModel() {
+    self.navigationBarView.rac.hidden = self.viewModel.outputs.navigationBarIsHidden
```

Comment:
In the view controller we are calling the method `setupNavigationView` which constraints the `navigationBarView` to `navigationController.navigationBar`. When tapping on comments or updates we want the default navigationBar (back button and title) instead of our custom view. 

If we don't hide this conditionally the navigation bar will perform a strange behavior when calling `pushViewController` -- the default will show initially and then quickly be overridden by our custom navigation bar view. Hiding will ensure that we are only using the default when pushing but still using the custom otherwise.